assets/js/legacy/myparcelbe.js

```diff
@@ -0,0 +1,805 @@
+MyParcelBE = {
+  /*
+     * Init
+     *
+     * Initialize the MyParcel BE checkout.
+     *
+     */
```

Comment:
Command staat niet helemaal recht, controleer ook de andere commands in dit bestand